pkg/api/connection/connection.proto

```diff
@@ -57,6 +56,8 @@ message Connection {
   State state = 9;
 }
 
+message Metrics { map<string, string> metrics = 1; }
```

Comment:
How had we previously been handling Metrics?

Reply:
In CrossConnectEvent only.

Reply:
Hmm... so... could it make sense to attach Metrics to PathSegments rather than here?

Reply:
Previously they was for Cross connection between Client <-> Endpoint, so I think they should be only in event, since in case of metrics we will not send update to connection object itself.

Reply:
There are tradeoffs there... if I wanted a full picture for the metrics along the path I'd have to be able to get the metrics from every stage in the path and correlate them.  If we put Metrics with PathSegments, then I can get Metrics for the complete Path from my Client and see exactly what seems to be breaking down where.

Reply:
This make sense indeed, but in this case it need to be a bit more structured instead of just map of values.
Let me think today on proposal and I will update this PR with it.

Reply:
Might also be good to queue up alternatives to discuss at tomorrows community meeting :)